target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-r619ac-128m.dts

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-r619ac.dtsi"
+
+/ {
+	model = "P&W R619AC 128M";
+	compatible = "p2w,r619ac-128m";
+
+	chosen {
+		bootargs-append = " root=/dev/ubiblock0_1 rootfstype=squashfs";
+	};
+};
+
+&rootfs_part1 {
+	reg = <0x0 0x8000000>;
+};
+
+/delete-node/ &rootfs_part2;
```

Comment:
I would rather not delete and change stuff here, but just put the nand setup into device DTS files for both cases.

Reply:
Ping.